### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Axel Antonin Constant Pittet and Adam Kurt Gruber

### DISCUSSION THREAD

**Discussion by AdamoElProfesor** [heig-vd-dai-course/heig-vd-dai-course#356] [Show and tell]
Practical work 1

Create a CLI that translates an English text file to different languages.

**Languages to translate to**:

- French
- Spanish
- Italian
- German
- Simplified Chinese if time allows it (encode is different)

**Library to use**: <https://github.com/DeepLcom/deepl-java>

**Supported languages**: <https://developers.deepl.com/docs/resources/supported-languages>

**Comment by ludelafo** [comment]
Looks good to us, thank you!

**Comment by Axwells** [comment]
We finished the practical work, it is now available on our repository under this link : https://github.com/Axwells/DAI_PracticalWork1
